src/app/services/seo.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
+import { Title, Meta } from '@angular/platform-browser';
+
+@Injectable()
+export class SeoService {
+  // This part is happended at the end of head>title
+  private siteTitle = 'RxJS Documentation';
+
+  // head>meta>description for common pages
+  public homeDescription = 'The complete RxJS documentation...';
+  public operatorsDescription = 'All the RxJS operators...';
+  public companiesDescription = 'Companies that use RxJS...';
+  public teamDescription = 'People behind the RxJS Documentation project...';
```

Comment:
Would it be better to utilize an const Object for page seo meta data, something like 

```
{ 
     home : { 
         title : ' lorum ipsum',
         description: '', 
      },  
      .....
}

```

This way we can access it per page

Reply:
Why not store this data in the route config and listen to router events to update the title and SEO instead of setting it from each component?

Reply:
I agree, this seems ideal. ☝️

Reply:
Do you agree that I make it the way it is done in this sample app, defining the data in the route config directly (via an external file if we want to facilitate translations) and processed with guards:

https://github.com/feloy/ng-universal-demystified/blob/master/src/app/app.module.ts

Reply:
I agree with using the route config but don't think it needs to be processed with guards. You can get the activated route from the route tree in the NavigationEnd event.

Here is an plunker: http://plnkr.co/edit/6iVjZcnqKBi7Dpp3KrFX?p=info 

The `components.ts` has the example code.